src/lib.rs

```diff
@@ -8,11 +8,24 @@
 //! If it's too slow (e.g.  e.g. all/some leds are white or display the wrong color)
 //! you may want to try the `slow` feature.
 
+// https://wp.josh.com/2014/05/13/ws2812-neopixels-are-not-so-finicky-once-you-get-to-know-them/
+//                                          Min     Typ     Max     Units
+// T0H      0 code  high voltage time       200     350     500     ns
+// T1H      1 code  high voltage time       550     700     5,500   ns
+// TLD      data    low voltage time        450     600     5,000   ns
+// TLL      latch   low voltage time        6,000                   ns
+
+// The tricky timing is the T0H between 200 and 350ns timing.
+// 3mhz is 333ns per block and should be flexible for all other timings
+
+// But new device reset latch timing is posted as 280us
+// https://blog.particle.io/2017/05/11/heads-up-ws2812b-neopixels-are-about-to-change/
+
 #![no_std]
 
 use embedded_hal as hal;
 
-use crate::hal::digital::OutputPin;
+use crate::hal::digital::v2::OutputPin;
```

Comment:
Please do this change in a separate pr, we might also want to add (configurable) error handling

Reply:
Done in #2